public/app/core/components/AppChrome/History/HistoryWrapper.tsx

```diff
@@ -78,127 +82,205 @@ function HistoryEntryAppView({ entry, isSelected, onClick }: ItemProps) {
-                  store.setObject('CLICKING_HISTORY', true);
-                  onClick();
-                }}
-                isCompact={true}
-                className={view.time === selectedViewTime ? undefined : styles.card}
-              >
-                <Stack direction="column" gap={0}>
-                  <Text variant="bodySmall">{view.name}</Text>
-                  {view.description && (
-                    <Text color="secondary" variant="bodySmall">
-                      {view.description}
-                    </Text>
-                  )}
-                </Stack>
-              </Card>
-            );
-          })}
-        </div>
-      )}
-    </Stack>
+    </span>
   );
 }
 const getStyles = (theme: GrafanaTheme2) => {
   return {
     card: css({
+      label: 'card',
+      background: 'none',
+      margin: theme.spacing(0.5, 0),
+    }),
+    cardSelected: css({
+      label: 'card-selected',
+      background: 'none',
+    }),
+    subCard: css({
+      label: 'subcard',
       background: 'none',
+      margin: 0,
     }),
     iconButton: css({
+      label: 'expand-button',
       margin: 0,
     }),
+    iconButtonCircle: css({
+      label: 'blue-circle-icon',
+      margin: 0,
+      background: theme.colors.background.primary,
+      fill: theme.colors.primary.main,
+      cursor: 'default',
+      '&:hover:before': {
+        background: 'none',
+      },
+      //Need this to place the icon on the line, otherwise the line will appear on top of the icon
+      zIndex: 0,
+    }),
+    iconButtonDot: css({
+      label: 'blue-dot-icon',
+      margin: 0,
+      color: theme.colors.primary.main,
+      border: theme.shape.radius.circle,
+      cursor: 'default',
+      '&:hover:before': {
+        background: 'none',
+      },
+      //Need this to place the icon on the line, otherwise the line will appear on top of the icon
+      zIndex: 0,
+    }),
     expanded: css({
+      label: 'expanded',
       display: 'flex',
       flexDirection: 'column',
-      marginLeft: theme.spacing(5),
+      marginLeft: theme.spacing(6),
       gap: theme.spacing(1),
       position: 'relative',
       '&:before': {
         content: '""',
         position: 'absolute',
-        left: theme.spacing(-2),
+        left: 0,
         top: 0,
         height: '100%',
         width: '1px',
         background: theme.colors.border.weak,
       },
     }),
+    borderLeft: css({
+      label: 'border-left',
+      position: 'relative',
+      height: '100%',
+      width: '100%',
+      paddingLeft: theme.spacing(2),
+      '&:before': {
+        content: '""',
+        position: 'absolute',
+        left: theme.spacing(5.75),
+        top: 0,
+        height: '100%',
+        width: '1px',
+        background: `repeating-linear-gradient(
+          to bottom,
+          ${theme.colors.border.strong},
+          ${theme.colors.border.strong} 2px,
+          transparent 2px,
+          transparent 4px
+        )`,
```

Comment:
if this is for the dashed timeline, can we use border and `border-style: dashed` instead?